applications/fax/src/fax_worker.erl

```diff
@@ -908,6 +920,41 @@ convert_openoffice_document(CT, TmpDir, JobId, RespContent) ->
             {'error', <<"can not convert file, try uploading a tiff">>}
     end.
 
+-spec convert_document_convertapi(kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary(), kz_term:ne_binary()) ->
+                                         {'ok', kz_term:ne_binary()} |
+                                         {'error', kz_term:ne_binary()}.
+convert_document_convertapi(CT, TmpDir, JobId, RespContent, Secret) ->
+    Extension = kz_mime:to_extension(CT),
+    FileName = list_to_binary([JobId, ".", Extension]),
+    RequestURL = list_to_binary([?CONVERTAPI_URL, "/", Extension, "/to/tiff?ImageResolutionH=204&ImageResolutionV=392&ScaleImage=true&ScaleProportions=true&ScaleIfLarger=true&ImageHeight=4312&ImageWidth=1728&Secret=", Secret]),
+    Boundary = kz_term:to_hex_binary(crypto:strong_rand_bytes(24)),
+    ContentTypeHeaderValue = list_to_binary(["multipart/form-data; boundary=", Boundary]),
+    Headers = [{"Content-Type", ContentTypeHeaderValue}],
+    Body = list_to_binary(["--", Boundary
```

Comment:
Does `Body` need to be a `binary()`? Or can it remain an `iolist()` into `kz_http`?